content/dev/dev-guide.md

```diff
@@ -0,0 +1,264 @@
+
+## Pre-requisites
+
+### Homebrew
+- Homebrew is a package manager for macOS, we'll need it installed in order to install `git-lfs`.
+- ```/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"```
+
+### Git LFS
+- A package that enables git to work with large files, we'll install this via brew.
+- ```brew install git-lfs```
+
+### Wget
+ - A package used to download artifacts from the web
+ - `brew install wget`
+
+### Visual Studio Code
+ - While not technically a prerequisite, there's a hoon syntax highlighting plugin available and this is the editor we recommend: https://code.visualstudio.com/
+ - For the small text file edits in this guide, we'll simply use `nano`.
+
+### Urbit Install
+ - First we need to create a directory for the urbit binary, we'll create this in your home directory.
+   - ```bash
+     mkdir ~/urbit-dev
+     ```
+ - Next we're going to pull down the latest urbit binary from urbit.org from the newly created directory.
+    - ```bash
+      cd ~/urbit-dev
+      curl -JLO https://urbit.org/install/mac/latest
+      ```
+  - Now we'll untar the contents of the download into our newly created directory.
+    - ```bash 
+      tar zxvf ./darwin.tgz --strip=1
+      ```
+  - With the urbit binary installed we're ready for the basic dev workflow.
+
+## Development Basics
+
+### Fake Ships
+#### What is a "fake" ship?
+ - For local development we'll create a 'fake' ship, this is a local ship that is not connected to the urbit network. This means we don't need a real ID (eg comet, planet) and can delete and recreate the ship whenever we want to reset things.
+#### Creating a fake ship
+ - We can create a local fake ship with the below command. This will create a fake ship named zod in our `urbit-dev` directory. This will take a couple of minutes and generate some output in the terminal.
+   - ```bash
+     ./urbit -F zod
+     ```
+ - When it's finished you'll see a command prompt like `~zod:dojo>` - this is the urbit dojo where you can enter and run urbit commands.
+ - To exit the dojo type `|exit`
+#### Fake ship networking
+ - Fake ships on the same machine can automatically talk to each other. Let's create a second fake ship named `~bus` and confirm this.
+   - ```bash
+     ./urbit -F bus
+     ```
+ - Now you'll have two local ships, `~zod` and `~bus`.
+ - From the newly created `~bus` dojo enter `|hi ~zod` and you'll see that urbit's networking finds the first fake ship we created.
+ - Exit the `~bus` dojo with `|exit`
+
+#### Faster fake ship booting
+A common part of the development workflow is deleting and creating fake ships. The couple of minutes this takes slows down development. Instead we can create a backup copy of a freshly created fake ship. This way when you want to delete some fake ship you've been working on and reset it, it can be done instantly rather than having to wait for a new one to be created.
```

Comment:
This is good to know and I think changes my mind about this?

If the in the common case you don't have to boot a new fake ship that often, then we could reduce a lot of the guide (move this into an advanced tips and tricks section rather than the default).